docs/configuration/settings/features/remote-control.md

```diff
@@ -1,6 +1,8 @@
 ## Remote Control
 
-Secure remote capabilities are offered for Trio using the *LoopFollow* app with these versions:
+The use of Nightscout Careportal and [Shortcuts](shortcuts.md){: target="_blank" } continue to be supported with Trio.
```

Comment:
Please don't add this blank statement, but specify which entries work (carb correction, temp target). Users coming from iAPS may still expect this to work the way they are used to with the weird announcements for remote commands.

I suggest something like `The *limited* use of Nightscout Careportal*, i.e., entry of Carb Correction, Temporary Targets,* ...`

Reply:
I revised the Nightscout words and then removed the shortcuts link - it has it's own page and didn't really fit here.